### PR TITLE
🧹Drop deprecated fields in azure.sql.server resource.

### PR DESCRIPTION
**Pull request by preslavgerchev**
Drop deprecated azure fields that are no longer used

### CODE REVIEW

**Comment by tas50** [comment]
Can we make sure to wait for v8 for this one

**Comment by preslavgerchev** [comment]
> Can we make sure to wait for v8 for this one

Sounds good. I will fix the title a bit so we wait on it

File: resources/packs/azure/azure.lr [hunk 1]
```diff
@@ -834,10 +828,6 @@ private azure.subscription.sqlService.database @defaults("id name") {
   elasticPoolName string
   // SQL database default secondary location
   defaultSecondaryLocation string
-  // Deprecated
-  serviceTierAdvisors() dict
-  // Deprecated. Use `advisor.recommendedActions` instead.
```
Comment (vjeffrey) [inline resources/packs/azure/azure.lr:839]: none of our policies reference these fields, right?

**Review by vjeffrey — APPROVED** [review]

**Comment by tas50** [comment]
Before we pull the trigger here can we double check on the process we wanted to take for deprecation. I opened a ticket to get formal deprecation going and Dom had mentioned removing this all in v9 after we have a deprecation going

**Comment by tas50** [comment]
If it's broken we should go ahead and remove it. If a user could use it then we should deprecate it so they know to update.

File: resources/packs/azure/azure.lr (hunk 1 above)
Comment (preslavgerchev) [reply resources/packs/azure/azure.lr:839]: nope, these fields have been like this (not returning a value, only errors) for some time now.
I also double checked the policies so we're good

**Comment by preslavgerchev** [comment]
I am going to go ahead and merge that, these fields havent been working for a while now so I think we're safe to cut them

**Pull request merged**